sql/SQLCode.sql

```diff
@@ -0,0 +1,26 @@
+CREATE User(
+user_ID VARCHAR(5),
+u_name VARCHAR(30) NOT NULL,
+u_pwd VARCHAR(30) NOT NULL,
+is_admin BIT DEFAULT 0,
+PRIMARY KEY (user_ID)
+);
+
+CREATE MetaData(
+ID VARCHAR(5),
+file_size DECIMAL(4,0) NOT NULL,
+date_added DATE,
```

Comment:
I suggest we use int here, because then we can just use the unix timestamp which is just the number of seconds since 1970.
Then we don't have to crazy parsing and stuff to get a proper datetime

```suggestion
date_added INT,
```